pom.xml

```
@@ -76,8 +78,7 @@
     <bnd.includeresource>-${.}/NOTICE, -${.}/*.xsd</bnd.includeresource>
 
     <feature.directory>src/main/feature/feature.xml</feature.directory>
-    <spotless.check.skip>true</spotless.check.skip> <!-- Spotless disabled for now -->
-    <spotless.version>1.31.1</spotless.version>
+    <spotless.version>2.0.0</spotless.version>
```

There are also a few newer versions nowadays, see [changelog](https://github.com/diffplug/spotless/blob/master/plugin-maven/CHANGES.md):

```suggestion
    <spotless.version>2.0.3</spotless.version>
```